src/test/java/com/openlattice/rehearsal/datasets/DataSetsTest.kt

```diff
@@ -0,0 +1,107 @@
+package com.openlattice.rehearsal.datasets
+
+import com.openlattice.organization.OrganizationIntegrationAccount
+import com.openlattice.organizations.Organization
+import com.openlattice.rehearsal.authentication.MultipleAuthenticatedUsersBase
+import junit.framework.Assert
+import org.junit.BeforeClass
+import org.junit.Test
+import java.sql.DriverManager
+import java.sql.SQLException
+import java.util.*
+
+
+open class DataSetsTest : MultipleAuthenticatedUsersBase() {
+    companion object {
+        lateinit var credentials: OrganizationIntegrationAccount
+        lateinit var organization: Organization
+        lateinit var organizationID: UUID
+        lateinit var url: String
+
+        const val create_query = "CREATE TABLE IF NOT EXISTS " +
+                "authors (id serial PRIMARY KEY, name VARCHAR(25));"  +
+                "INSERT INTO authors(id, name) VALUES(1, 'Jack London');"
+        val delete_column_query = "ALTER TABLE authors " +
+                "DROP COLUMN name;"
+        val delete_table_query = "DROP TABLE authors;"
+        val tableName = "authors"
+        val columnName = "name"
+
+        @JvmStatic
+        @BeforeClass
+        fun init() {
+            loginAs("admin")
+
+            organization = createOrganization()
+            organizationID = organization.id
+//            organizationID = UUID(1L, 0L)
+            url = "jdbc:postgresql://localhost:5432/org_" + organizationID.toString().replace("-", "")
+            credentials = organizationsApi.getOrganizationIntegrationAccount(organizationID)
+       }
+
+    }
+
+    fun uploadData() {
+        val id = 1
+        val author = "Jack London"
+
+        try {
+            DriverManager.getConnection(url, credentials.user, credentials.credential).use { con ->
```

Comment:
This should use a `HikariDataSource` to connect to postgres instead of DriverManager. Looks like a lot of the logic to handle this was built already in `TestAssemblerConnectionManager`, so you should be able to call `TestAssemblerConnectionManager.connect(organizationID, properties)` (where properties is where the username/credential are specified) which will return a HikariDataSource, and that's where you should pull your connections from. (also no need to build the jdbc string with this approach)

Check out `AssemblerLinkingTest`. lines 831-853 for an example of how to do this